src/test/java/Homework2.java

```diff
@@ -0,0 +1,117 @@
+         //Assert.assertEquals(expectedValue,actualValue,Message);
+
+
+         By clickLoginBtnLocator = By.id("loginbutton");
+         WebElement clickLoginBtn = driver.findElement(clickLoginBtnLocator);
+         clickLoginBtn.click();
+
+         // keep this in method to access when u need.
+         try {
+             Thread.sleep(2000);
+         } catch (InterruptedException e) {
+             e.printStackTrace();
+         }
+
+         // driver.findElement(By.id("loginbutton")).click();
+
+         By isErrorMessageDisLoc = By.xpath("(//form/div)[1]");
+         WebElement isErrorMessageDis = driver.findElement(isErrorMessageDisLoc);
+         boolean isDisplayed = isErrorMessageDis.isDisplayed();
+         Assert.assertTrue(isDisplayed,"Test Failed");
+
+         By isContinueLocator = By.xpath("//button[@id= 'loginbutton']");
+         WebElement isContinueEnabled = driver.findElement(isContinueLocator);
+         boolean isEnabled = isContinueEnabled.isEnabled();
+         Assert.assertTrue(isEnabled, "Test Failed");
+
+         // very last step, just to close your browser window
+         driver.quit();
+     }
+
+   /* Testcase-2:
+     On Create new account page:
+    Verify the "Sign Up" button is enabled when user lands on the form
+             * Enter the below details in Sign Up form EXCEPT DOB
+     * First name
+     * Last name
+     * email
+     * re-enter email
+     * new password
+     * click Sign Up
+     *
+     * Verify "Please choose a gender. You can change who can see this later." is displayed */
+     @Test
+    public void createNewAccountPage() {
+        System.setProperty("webdriver.chrome.driver", "./Drivers/chromedriver");
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://www.facebook.com/");
+
+        driver.findElement(By.xpath("(//*[@role = 'button'])[2]")).click();
+        try {
+            Thread.sleep(2000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+        driver.findElement(By.xpath("//button[text() = 'Sign Up']")).isEnabled();
+        boolean isItEnabled = driver.findElement(By.xpath("//button[text() = 'Sign Up']")).isEnabled();
+        Assert.assertTrue(isItEnabled,"Testcase failed button is not enabled");
+        driver.findElement(By.xpath("//input[@name='firstname']")).sendKeys("Dalena");
+        driver.findElement(By.xpath("//input[@name='lastname']")).sendKeys("Eyasu");
+        driver.findElement(By.xpath("//input[@name='reg_email__']")).sendKeys("dalenatechno@yahoo.com");
+        driver.findElement(By.xpath("//input[@name='reg_email_confirmation__']")).sendKeys("dalenatechno@yahoo.com");
+        driver.findElement(By.xpath("(//input[@ type ='password'])[2]")).sendKeys("technosoft");
+        driver.findElement(By.xpath("//button[text() = 'Sign Up']")).click();
+        boolean isDisplayed= driver.findElement(By.id("js_ex")).isDisplayed();
```

Comment:
`js_ex` also not a reliable value this is a random value, may change any moment and your test gonna fail. In this case you can use text() to create xpath.